src/pages/SignIn/SignIn.js

```diff
@@ -49,10 +50,10 @@ const SignIn = ({ token }) => {
   const steps = ['Visitor Information', 'Picture Time', 'Capture'];
 
   const handleSubmit = values => {
-    console.log(values);
     async function _handleSubmit() {
+      const slackPicUrl = await sendPic(values.picture);
       const response = await signInService.checkInVisitor(
-        values,
+        { ...values, url_private: slackPicUrl },
```

Comment:
would we need to check `const slackPicUrl = await sendPic(values.picture);` if this passes then only move on? other than that +1